pkg/minikube/cluster/cluster.go

```diff
@@ -152,15 +152,15 @@ func UnRegister(api libmachine.API) (bool, error) {
 		return false, err
 	}
 
-	if drivers.MachineInState(host.Driver, state.Running)() {
-		err := registration.UnregisterHostVM(host, RegistrationParameters)
-		if err != nil {
-			return false, err
-		}
-	} else {
-		return true, nil
+	if !drivers.MachineInState(host.Driver, state.Running)() {
+		return true, fmt.Errorf("Unable to verify registration. VM not in running state")
```

Comment:
So why _true_? The VM is not running, so how do you know that it "supports" unregistration.